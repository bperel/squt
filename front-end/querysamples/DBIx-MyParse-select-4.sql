SELECT * FROM table1 USE INDEX (index1, index2)
