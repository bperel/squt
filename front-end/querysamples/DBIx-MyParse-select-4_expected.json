{
	"Tables":{
		"table1":{
			"table1":{
				"OUTPUT":{
					"*":{
						"-1":"*"
					}
				}
			}
		}
	}
}
